peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -51,6 +51,11 @@ <h1>{{ document.title }}</h1>
                  target="_blank">
                 <i class="bi bi-linkedin linkedin-forecolor share-icon"></i>
               </a>
+              <a href="mailto:example@exapmle.com?subject=Check out this document&body={{ request.build_absolute_uri }}"
```

Comment:
```suggestion
              <a href="mailto:?subject=Take a look at this document from {{ APP_NAME }}: {{ document.title|truncatewords:7|escape }}&body={{ request.build_absolute_uri }}"
```

Please also make that "Take a look at this document from {{ APP_NAME }}" bit translateable.